allennlp/models/semantic_role_labeler.py

```diff
@@ -191,8 +198,43 @@ def get_viterbi_pairwise_potentials(self):
 
     @classmethod
     def from_params(cls, vocab: Vocabulary, params: Params) -> 'SemanticRoleLabeler':
-        text_field_embedder = TextFieldEmbedder.from_params(vocab, params.pop("text_field_embedder"))
-        stacked_encoder = Seq2SeqEncoder.from_params(params.pop("stacked_encoder"))
+        """
+        With an empty ``params`` argument, this will instantiate a SRL model with the same
+        configuration as published in the Deep Semantic Role Labeling - What works and what's
+        next  paper, as long as you've set ``allennlp.common.constants.GLOVE_PATH`` to the
+        location of your gzipped 100-dimensional glove vectors.
+
+        If you want to change parameters, the keys in the ``params`` object must match the
+        constructor arguments above.
+        """
+        default_embedder_params = {
```

Comment:
Something like that would be good. We also need to consider doing something like they have in `torchvision` where a pretrained model is autodownloaded and saved somewhere if you try to use a model, that would be nice.